README.md

```diff
@@ -27,115 +31,164 @@ repositories' pull requests by using the GitHub Pull Request Builder plugin. It
  - Create administrative jobs
- - Create slaves credentials
- - Create GitHub credentials
- - Create slave nodes
- - Create Host OS builds jobs
+ - Setup SSH credentials used to access Jenkins slaves from Jenkins master
+ - Create credentials used to access GitHub API from Jenkins
+ - Create Jenkins slaves in Jenkins web UI
+ - Create builds jobs
+
 They are described in detail below.
 
-### Prepare CentOS system
+### Prepare CentOS system(s)
+
+First, you will need to prepare at least one CentOS system. If the Jenkins
+instance will be used for testing, one system is enough; production instances
+should use more systems.  Currently, OpenPOWER Host OS has 1 Jenkins slave for
+validating the code from GitHub pull requests and 3 Jenkins slaves for builds.
+
+KVM virtual machines are recommended, but you could optionally use bare-metal or
+containers instead. If KVM virtualization is used, for each required system,
+you will need to create a KVM guest and install a CentOS ISO in it. It is
+recommended to use virt-install to do it.
+
+If you do not have a direct network connection to the system(s), you may
+setup a SSH proxy in ~/.ssh/config in your workstation to make the SSH
+proxy transparent. Example:
 
-First, you'll need to prepare at least one CentOS system. Virtual machines are
-recommended, but they could also be bare-metal or containers.
+Host ophostos-builds
+    HostName <target_server>
+    ProxyCommand ssh <myuser>@<proxy_server> nc <target_server> 22
 
-Note: to be able to upload the results of builds to a server and push commits to
-GitHub automatically, the slaves must have be able to identify the hosts. Add
-them to the "known_hosts" file by executing a simple ssh command from the slaves
-to the upload server and to github.com. Alternatively, you could disable
-"StrictHostKeyChecking", but that may be a security hazard.
+After the SSH proxy is configured, simply use the following to connect to the system:
+`ssh <myuser>@ophost-builds`
 
-### Execute Ansible playbooks
+### Setup SSH credentials used to access upload server and GitHub from Jenkins slaves
 
-There are Ansible playbooks that set up the Jenkins master node and two Jenkins
-slave nodes, one for Host OS builds and one for validating the code from GitHub
-pull requests. There could be multiple slave nodes set up for parallel builds.
-The different roles (master, builds slave and validation slave) could even be
-set up in the same system, as desired.
+To automate SSH authentication between 2 hosts, it is necessary to:
+ - create a SSH key pair if you still do not have one
+ - add the target host to ~/.ssh/known_hosts in the source host. It is also
+   possible to do it by executing a simple SSH command from the source host to
+   the target host. Alternatively, you could disable "StrictHostKeyChecking"
+   for all SSH connections in the source host, but that may be a security hazard.
+ - add the SSH private key to ~/.ssh/ in the source host
```

Comment:
This step is actually done by the ansible playbook, but I see no harm in leaving it here too.